### PR TITLE
Add MediaVision explanation in overview.md of what-is-tizen

### PR DESCRIPTION
**Pull request by tyoungroy**
Signed-off-by: Tae-Young Chung <ty83.chung@samsung.com>

### CODE REVIEW

File: docs/platform/what-is-tizen/overview.md [hunk 1]
```diff
@@ -46,6 +46,13 @@ The most important strengths that enable Tizen to move to the next level are IoT
 
   ![IoTivity for Connectivity](media/about_tizen_6.png)
 
+## AI and Machine Learning
+
+MediaVision  provides task based inference API; classification, detection, and pose landmark detection.
+It also provides easy ways for a user:
+- to run models with meta files which describe model information
+- to run various type of models with Machine Learning API
+- to use H/W acceleration with NNRuntime
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/overview.md:55]: The first word of a sentence should always be capitalized. Please look into this issue.
```suggestion
- To use H/W acceleration with NNRuntime
```

File: docs/platform/what-is-tizen/overview.md [hunk 2]
```diff
@@ -46,6 +46,13 @@ The most important strengths that enable Tizen to move to the next level are IoT
 
   ![IoTivity for Connectivity](media/about_tizen_6.png)
 
+## AI and Machine Learning
+
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/overview.md:50]: "MediaVision" should be written as a third heading to make it distinguishable from other third headings under the "AI and Machine Learning" section.
Ref: In other PRs (https://github.com/Samsung/tizen-docs/pull/1616 & https://github.com/Samsung/tizen-docs/pull/1611), different authors are working on different sections under the "AI and Machine Learning" section.
```suggestion
### MediaVision
```

File: docs/platform/what-is-tizen/overview.md [hunk 3]
```diff
@@ -46,6 +46,13 @@ The most important strengths that enable Tizen to move to the next level are IoT
...
+MediaVision  provides task based inference API; classification, detection, and pose landmark detection.
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/overview.md:51]: An extra space has been removed.
```suggestion
MediaVision provides task based inference API; classification, detection, and pose landmark detection.
```

File: docs/platform/what-is-tizen/overview.md [hunk 4]
```diff
@@ -46,6 +46,13 @@ The most important strengths that enable Tizen to move to the next level are IoT
...
+It also provides easy ways for a user:
+- to run models with meta files which describe model information
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/overview.md:53]: The first word of a sentence should always be capitalized. Please look into this issue.
```suggestion
- To run models with metafiles which describe model information
```

File: docs/platform/what-is-tizen/overview.md [hunk 5]
```diff
@@ -46,6 +46,13 @@ The most important strengths that enable Tizen to move to the next level are IoT
...
+- to run various type of models with Machine Learning API
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/overview.md:54]: The first word of a sentence should always be capitalized. Please look into this issue.
```suggestion
- To run various types of models with Machine Learning API
```

**Review by omar-srbd — COMMENTED** [review]
@tyoungroy Hello,
I have reviewed only the updated part of the file on this PR and suggested some changes.
Please look into them.

**Comment by journey2w** [comment]
@asm-foysal 
Please review this PR after releasing Tizen Studio 4.6.

**Review by omar-srbd — COMMENTED** [review]
Hello @tyoungroy,  
Hope you are doing well. Since this PR has been labeled urgent, please incorporate the changes at your earliest convenience. We are hoping to merge this PR as soon as possible. Thanks.

**Review by omar-srbd — APPROVED** [review]
@tyoungroy, thanks for incorporating the changes on a timely manner.

@asm-foysal @mijin85cho I am approving this PR.
Please proceed with merging of this PR.

File: docs/platform/what-is-tizen/overview.md [hunk 6]
```diff
@@ -77,6 +77,13 @@ Compiler Toolchain
    - Neural network model quantizer.
    - Various profiles and test scripts to evaluate performance on target.
 
+### MediaVision
+MediaVision provides task based inference API; classification, detection, and pose landmark detection.
+It also provides easy ways for a user:
+- To run models with metafiles which describe model information
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/overview.md:83]: Please remove the "to" from the beginning of the sentence.
```suggestion
- Run models with metafiles which describe model information
```

File: docs/platform/what-is-tizen/overview.md [hunk 7]
```diff
@@ -77,6 +77,13 @@ Compiler Toolchain
...
+- To run various types of models with Machine Learning API
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/overview.md:84]: Please remove the "to" from the beginning of the sentence.
```suggestion
- Run various types of models with Machine Learning API
```

File: docs/platform/what-is-tizen/overview.md [hunk 8]
```diff
@@ -77,6 +77,13 @@ Compiler Toolchain
...
+- To use H/W acceleration with NN Runtime
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/overview.md:85]: Please remove the "to" from the beginning of the sentence.
```suggestion
- Use H/W acceleration with NN Runtime
```

File: docs/platform/what-is-tizen/overview.md [hunk 9]
```diff
@@ -77,6 +77,13 @@ Compiler Toolchain
    - Neural network model quantizer.
    - Various profiles and test scripts to evaluate performance on target.
 
+### MediaVision
+MediaVision provides task based inference API; classification, detection, and pose landmark detection.
+It also provides easy ways for a user:
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/overview.md:82]: When a colon introduces a list, the text that precedes the colon must be able to stand alone as a complete sentence.
Ref: https://developers.google.com/style/colons#introductory-phrase-preceding-colon
```suggestion
It also provides an easy way for a user to do the following:
```

**Review by omar-srbd — COMMENTED** [review]
@tyoungroy, there are a few changes to address.
Please check the Github suggestions that I have made to find the changes and implement them at your earliest convenience.

**Comment by mijin85cho** [comment]
Sorry for the interrupt with other issue : 

Hello, @tyoungroy 

BTW, Would you please check below issue also?

https://github.com/Samsung/tizen-docs/issues/1610

An issue is reported under native/guides/multimedia/face-detection.md,
so I ask you to check.
If you're not the right person to check, would you please assign the right person to the issue?

**Review by omar-srbd — APPROVED** [review]
@tyoungroy, thanks for incorporating all the suggested changes.

@asm-foysal @mijin85cho I am approving this PR.
Please proceed with merging this PR.

**Pull request merged**